grakn-engine/src/main/java/ai/grakn/factory/GraphFactory.java

```diff
@@ -40,10 +40,14 @@ private GraphFactory() {
     }
 
     public synchronized GraknGraph getGraph(String keyspace) {
-        return FactoryBuilder.getFactory(keyspace, null, graphConfig).getGraph(false);
+        GraknGraph graph = FactoryBuilder.getFactory(keyspace, null, graphConfig).getGraph(false);
+        graph.showImplicitConcepts(true);
```

Comment:
In most cases yes. The only case where this will change may be visualiser controller. I think . . .